common/arrow/Cargo.toml

```diff
@@ -17,7 +17,8 @@ simd = ["arrow/simd"]
 # Github dependencies
 arrow = { package = "arrow2", git="https://github.com/datafuse-extras/arrow2", rev = "7765067" }
 arrow-flight = { git="https://github.com/datafuse-extras/arrow2", rev = "7765067" }
-parquet = {package = "parquet2", git = "https://github.com/datafuse-extras/parquet2", rev = "d28330f"}
+#parquet2 = {package = "parquet2", git = "https://github.com/datafuse-extras/parquet2", rev = "d28330f"}
 # Crates.io dependencies
+parquet2 = { version = "0.3", optional = false, default_features = false, features = ["stream"] }
```

Comment:
Arrow2 has parquet already https://github.com/datafuse-extras/arrow2/tree/main/src/io/parquet
Do we still need the parquet2 crate here?

Reply:
At present, we need `parquet2` to extract/manipulate parquet Statistic.

To avoid implicit parquet2 dependency in Cargo.toml, we may need to re-export those types from datafuse-extras/arrow2 ()

Reply:
Got, here import parquet2 crate makes sense.

Reply:
Yes, if you just want to interact with parquet without serialization and deserialization of pages, then this would also be my recommendation: add a new dependency.

I would probably bump it to 0.4 so that it is aligned with arrow2' dependency and reduce the binary size a bit (parquet2 is very small, though). [Other 2 benefits of 0.4](https://github.com/jorgecarleitao/parquet2/releases/tag/v0.4.0) are:

1. writer no longer needs `Seek`.
2. supports async writers via `AsyncWrite`.

Reply:
@jorgecarleitao
roger & thanks